res/skins/Deere/style.qss

```diff
@@ -299,23 +299,24 @@ WLibrary QLineEdit,
     background-color: #006596;
     }
     #LibraryBPMSpinBox::up-button {
-      image: url(skin:/icon/ic_lib_spinbox_up_white.svg) no-repeat;
+      image: url(skin:/../Deere/icon/ic_lib_spinbox_up_white.svg) no-repeat;
       }
     #LibraryBPMSpinBox::down-button {
-      image: url(skin:/icon/ic_lib_spinbox_down_white.svg) no-repeat;
+      image: url(skin:/../Deere/icon/ic_lib_spinbox_down_white.svg) no-repeat;
       }
 
 /* button in library "Preview" column */
 QPushButton#LibraryPreviewButton {
   background: transparent;
   margin: 0px;
-  padding: 0px;
+  padding: 3px;
   border: 1px solid transparent;
   border-radius: 2px;
 }
 
 QPushButton#LibraryPreviewButton:!checked {
-  image: url(skin:/icon/ic_library_preview_play.svg);
+  /* TODO ronso0 Restore once fixed lib linKIconThemes (vers. <..80) is in *ubuntu main repos */
+  /*image: url(skin:/../Deere/icon/ic_library_preview_play.svg);*/
```

Comment:
Does this case a regression for all?
If so I prefere to revert this. The same applies to the other todos.

Reply:
this is the only way to get the Preview icons undistorted and adpating to the column width (though not growing with the row height). It's the same for all skins now, and special button styles are not re-implemented, thus the TODOS

so Yes, a regression. with the fixed lib `image` should work as it did before, and I will revert that asap after the patched libs are in main repos.